enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+   that material changes can no longer be initiated, and pauses its activity. Node updates that have already
+   been initiated continue beyond the maintenance schedule window.
+1. Since not all nodes have been updated, the MCO continues to expose a metric informing the system of
+   pending changes.
+1. In the subsequent days, the cluster is scaled up to handle additional workload. The new nodes receive
+   the most recent, desired configuration.    
+1. On the first Saturday of the next month, the MCO resumes its work. In order to ensure that forward progress is
+   made for all nodes, the MCO will update nodes that have the oldest current configuration first. This ensures
+   that even if the desired configuration has changed multiple times while maintenance was not permitted,
+   no nodes are starved of updates. Consider the alternative where (a) worker-node updates required > 24h,
+   (b) updates to nodes are performed alphabetically, and (c) MachineConfigs are frequently being changed
+   during times when maintenance is not permitted. This strategy could leave nodes sorting last 
+   lexicographically no opportunity to receive updates. This scenario would eventually leave those nodes
+   more prone to version skew issues.
+1. During this window of time, all node updates are initiated, and they complete successfully. 
+
+#### Service Delivery Emergency Patch
+1. SRE determines that a significant new CVE threatens the fleet.
+1. A new OpenShift release in each z-stream fixes the problem.
+1. SRE plans to override customer maintenance schedules in order to rapidly remediate the problem across the fleet.
+1. The new OpenShift release(s) are configured across the fleet. Clusters with permissive maintenance 
+   schedules begin to apply the changes immediately.
+1. Clusters with change management policies precluding updates are SRE's next focus.
+1. During each region's evening hours, to limit disruption, SRE changes the `changeManagement` strategy 
+   field across relevant resources to `Disabled`. Changes that were previously pending are now
+   permitted to be initiated. 
+1. Cluster operators who have alerts configured to fire when there is no change management policy in place 
+   will do so.
+1. As clusters are successfully remediated, SRE restores the `MaintenanceSchedule` strategy for its resources.
+   
+
+#### Service Delivery Immediate Remediation
+1. A customer raises a ticket for a problem that is eventually determined to be caused by a worker-node system configuration.
+1. SRE can address the issue with a system configuration file applied in a MachineConfig.
+1. SRE creates the MachineConfig for the customer and provides the customer the option to either (a) wait until their
+   configured maintenance schedule permits the material change from being initiated by the MachineConfigOperator
+   or (b) having SRE override the maintenance schedule and permitting its immediate application.   
```

Comment:
I don't expect SRE will override the schedule in this scenario.  The change would be staged and customer notified of options.  Customer can either wait for their next update window or the customer can permit the change to progress.